web/tsconfig.node.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.node.tsbuildinfo",
```

Comment:
Is this option required? Seems to reference a temporary file

Reply:
removed